packages/site/src/data/angular/courses.ts

```diff
@@ -53,4 +62,141 @@ export const courses: Course<typeof courseTags[number]>[] = [
 		href: "https://www.rxjs-fruits.com/subscribe",
 		tags: ["rxjs"],
 	},
+	{
+		title: "JavaScript Marathon: Using Angular Libraries in an Nx Monorepo",
```

Comment:
Let's just remove "JavaScript Marathon" from all titles - it's done inconsistently and detracts for the content title.